packages/devtools_app/pubspec.yaml

```diff
@@ -68,12 +68,13 @@ dependency_overrides:
   # See: https://github.com/flutter/devtools/pull/1342
   devtools_server:
     path: ../devtools_server
-
+  json_serializable: ^3.2.0
```

Comment:
is there a reason this is in dependency_overrides instead of dependencies?

Reply:
Yes, so its easier to debug with having to change the path in dependencies.